terralab/auth_helper.py

```diff
@@ -82,11 +82,11 @@ def get_tokens_with_custom_redirect(cli_config: CliConfig) -> tuple[str, str]:
     """
```

Comment:
can we update the comment on this method since we are no longer opening up the browser